frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -26,6 +44,10 @@ local ReaderZooming = InputContainer:new{
         contentheight = _("Zoom to fit content height works best with page view."),
         content = _("Zoom to fit content works best with page view."),
     },
+    panned_modes = {
+        column = _("In most cases, with column zoom mode, you'll want to set page view."),
```

Comment:
Maybe something like:
```suggestion
        column = _("Page view normally works best with column zoom mode."),
```